### PR TITLE
[pipeline/trusted-resources] Add presubmit-test.sh

### PR DESCRIPTION
**Pull request by wlynch**
# Changes

Adds presubmit-test.sh script to hook into project level CI.

Script stolen from
https://github.com/tektoncd/experimental/blob/main/pipeline-to-taskrun/test/presubmit-tests.sh,
with minor modifications to re-enable markdown linting and remove unused
environment variables.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [n/a] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Commit messages includes a project tag in the subject - e.g. [OCI], [hub], [results], [task-loops]

_See [the contribution guide](https://github.com/tektoncd/experimental/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by Yongxuanzhang** [comment]
the ci is failing but I cannot see any failures from the logs.

File: pipeline/trusted-resources/test/presubmit-tests.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# For convenience, it can also be executed manually.
+# Running the script without parameters, or with the --all-tests
+# flag, causes all tests to be executed, in the right order.
+# Use the flags --build-tests, --unit-tests and --integration-tests
+# to run a specific set of tests.
+
+# Markdown linting failures don't show up properly in Gubernator resulting
+# in a net-negative contributor experience.
+
+source $(dirname $0)/../../vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
```
Comment (Yongxuanzhang) [inline pipeline/trusted-resources/test/presubmit-tests.sh:27]: maybe this needs to be fixed

Comment (Yongxuanzhang) [inline pipeline/trusted-resources/test/presubmit-tests.sh:27]:
```suggestion
source $(dirname $0)/../../../vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
```
maybe like this?

Comment (wlynch) [reply pipeline/trusted-resources/test/presubmit-tests.sh:27]: Good catch!

**Comment by Yongxuanzhang** [comment]
I will fix the unit test

**Comment by wlynch** [comment]
> @Yongxuanzhang: changing LGTM is restricted to collaborators
> 
> In response to [this](https://github.com/tektoncd/experimental/pull/840#issuecomment-1057437142):
> 
> > /lgtm
> > /approve
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.

@Yongxuanzhang add yourself to https://github.com/tektoncd/community/blob/5bc525bd3681a4fedac09ba7bfe75c3b128e5c17/org/org.yaml#L509

**Comment by Yongxuanzhang** [comment]
> > @Yongxuanzhang: changing LGTM is restricted to collaborators
> > In response to [this](https://github.com/tektoncd/experimental/pull/840#issuecomment-1057437142):
> > > /lgtm
> > > /approve
> > 
> > 
> > Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md). If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
> 
> @Yongxuanzhang add yourself to https://github.com/tektoncd/community/blob/5bc525bd3681a4fedac09ba7bfe75c3b128e5c17/org/org.yaml#L509

thank you!

**Pull request merged**